src/objects/table/demo/basic.twig

```diff
@@ -0,0 +1,160 @@
+        <a href="https://github.com/cloudfour/pwastats/network/members">5</a>
+      </td>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">79</td>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">
+        <a href="https://github.com/cloudfour/pwastats/issues">7</a>
+      </td>
+    </tr>
+    <tr class="o-table__row">
+      <th class="o-table__heading" scope="row">
+        <a href="https://github.com/cloudfour/simple-svg-placeholder">simple-svg-placeholder</a>
+      </th>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">
+        <a href="https://github.com/cloudfour/simple-svg-placeholder/stargazers">89</a>
+      </td>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">
+        <a href="https://github.com/cloudfour/simple-svg-placeholder/network/members">4</a>
+      </td>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">89</td>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">
+        <a href="https://github.com/cloudfour/simple-svg-placeholder/issues">0</a>
+      </td>
+    </tr>
+    <tr class="o-table__row">
+      <th class="o-table__heading" scope="row">
+        <a href="https://github.com/cloudfour/SimpleSlideView">SimpleSlideView</a>
+      </th>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">
+        <a href="https://github.com/cloudfour/SimpleSlideView/stargazers">260</a>
+      </td>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">
+        <a href="https://github.com/cloudfour/SimpleSlideView/network/members">32</a>
+      </td>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">260</td>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">
+        <a href="https://github.com/cloudfour/SimpleSlideView/issues">6</a>
+      </td>
+    </tr>
+    <tr class="o-table__row">
+      <th class="o-table__heading" scope="row">
+        <a href="https://github.com/cloudfour/transition-hidden-element">transition-hidden-element</a>
+      </th>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">
+        <a href="https://github.com/cloudfour/transition-hidden-element/stargazers">34</a>
+      </td>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">
+        <a href="https://github.com/cloudfour/transition-hidden-element/network/members">0</a>
+      </td>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">34</td>
+      <td class="o-table__cell {% if numericCells %} o-table__cell--numeric {% endif %}">
+        <a href="https://github.com/cloudfour/transition-hidden-element/issues">0</a>
+      </td>
+    </tr>
```

Comment:
This isn't a blocker, but if you want to avoid all this repetition in the future, I believe Storybook includes Webpack's JSON loader by default, so you could put a file in the `demo` directory and pipe it to your Twig template to iterate over.

Reply:
Oh interesting I'll check that out

Reply:
Done! Let me know if there's a more storybooky way to do this!